_source/_posts/2020-11-23-quick-graphql-react-app.md

```diff
@@ -26,11 +26,9 @@ The first thing you need to do is sign up for an Everbase account and option an
 
 ## Set Up Authentication for Your React Application
 
-Next, you will need to set up a new Okta Application from your account administration page.  Log in to your developer's console and click **Applications**, and then click **Add Application**.  In the *Create New Application* section on the next page, select *Single Page App* and click **Next**.  On the *Application Settings* page, give your application a meaningful name.  I named mine *Exchange Rates*, but you can call yours whatever you want.  Because React will launch on port 3000, by default, you will want to change your URLs to `localhost:3000`.  Then click done.
+{% include setup/cli.md type="spa" framework="React" loginRedirectUri="http://localhost:3000/implicit/callback" %}
```

Comment:
Please remove "implicit" from the redirect. We try not to use it since the CLI creates authorization code flow apps.

Reply:
I will have to update the application code, otherwise it gives invalid_request

Reply:
I just noticed this repo doesn't have a GitHub repo. If you want, you can create one using the instructions in this tutorial. If you do that, please upgrade to the latest version of Okta React and the Sign-In Widget. I realize that might be difficult, so I can do it if you'd rather not.

Reply:
Just so you're aware, we need to upgrade any post that uses the Sign-In Widget to v5.2+. This is because of the Okta Identity Engine that's coming this summer. This is also the reason why we try not to use the Sign-In Widget in our posts. ;)

Reply:
This is my attempt for the migration: https://github.com/indiepopart/quick-graphql-react-app

Reply:
Looks great - thanks!